Workshop/Dependencies2.md

```diff
@@ -18,8 +18,8 @@ When upgrade is done, create 2 apps within the repository using the **Create a n
 | Name | `Common` |
 | Publisher | `<your publisher name>` |
 | ID Range (from..to) | `60000..60099` |
-| Include Sample Code | `N` |
-| Direct COMMIT | `Y` |
+| Include Sample Code | `no` |
+| Direct Commit | `yes` |
```

Comment:
```suggestion
| Direct Commit | `true` |
```

Generally, `true` and `false` in this file instead of `yes` and `no`?

Reply:
Yeah - the ALDoc PR contains a number of changes to the workshop - I will capture new images and change these strings as part of that repo (to avoid merging etc)